### PR TITLE
meeting notes for VM Summit #4

### PR DESCRIPTION
**Pull request by jasnell**
Closes: https://github.com/nodejs/vm/issues/6

### CODE REVIEW

File: meetings/2017-07-28.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        * Refine this criteria in a GH issue
+        * Opened https://github.com/nodejs/node/issues/14532
+        * Find two module owners and get buy in
+          * Matteo - https://github.com/mcollina/native-hdr-histogram will do an experimental release
+          * Work in issue #13254 to Identify the intersection of NAPI and async_hooks stability, which parts of the async_hooks are important to NAPI’s stability
+    * Take up a case study for Migrating NAN modules to N-API
+    * NAN Evolution
+ 
+* JerryScript use of N-API – Gabriel
+  * https://docs.google.com/presentation/d/1fE8g-r0Y7Xzin3LRn1fV6YBCFS5IcySjyRcARMSSadE/edit#slide=id.p3
```
Comment (cjihrig) [inline meetings/2017-07-28.md:57]: Can this be made available publicly?

Comment (jasnell) [reply meetings/2017-07-28.md:57]: Will ask Gabriel!

**Comment by refack** [comment]
Can I just land this now?

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**